tests/acceptance/basic-editor-test.js

```diff
@@ -214,7 +214,8 @@ test('keypress events when the editor does not have selection are ignored', (ass
 test('prevent handling newline', (assert) => {
   editor = Helpers.editor.buildFromText('', {element: editorElement});
 
-  editor.willHandleNewline(event => {
+  editor.willHandleNewline((event, {range}) => {
+    assert.equal(range.head.section.text, 'Line1', 'can access section text');
```

Comment:
This can be:
`assert.equal(editor.range.head.section.text, 'Line1', 'can access section text');`